apps/visualization/utils.py

```diff
@@ -278,6 +282,13 @@ def get_overview_table_data(
 
     existing_iso3 = Countries.objects.values_list('iso3', flat=True)
 
+    def format_table_data(data):
+        return {
+            'month': data['month'],
+            'indicator_value': data['indicator_value'],
+            'format': data['format'],
+        }
```

Comment:
Maybe we can directly use OverviewTableDataType instead.

Reply:
@navin we have extra attribute 'iso3' in qs, but we don't put iso3 in OverviewTableDataType

Reply:
hmm... okay so we can do this right?
```suggestion
    def format_table_data(data):
        return OverviewTableDataType(
            month=data['month'],
            indicator_value=data['indicator_value'],
            format=data['format'],
        )
```

Reply:
@bimal125 can you check this?

Reply:
format_table_data is used to ignore duplicated record of month so we should not do this

Reply:
format_table_data is further used to remove duplicated month so we don't do this change